bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -16,6 +22,16 @@
 
 public final class LaunchUtils {
 
+    private static final ILogger logger = Logger.getLogger(LaunchUtils.class);
+
+    private static final ServiceTracker<LaunchListener,LaunchListener> launchListeners;
+
+    static {
+        final BundleContext context = FrameworkUtil.getBundle(LaunchUtils.class).getBundleContext();
```

Comment:
It is in an exported package so any other bundle can import the package and use the class, even before the bndtools.core bundle is started. Maybe that does not happen today, but it could happen.